src/python/dxpy/scripts/dx.py

```diff
@@ -48,7 +48,7 @@
                            process_single_dataobject_output_args, find_executions_args, add_find_executions_search_gp,
                            set_env_from_args, extra_args, process_extra_args, DXParserError, exec_input_args,
                            instance_type_arg, process_instance_type_arg, process_instance_count_arg, get_update_project_args,
-                           property_args, tag_args, contains_phi, process_phi_param)
+                           property_args, tag_args, contains_phi, database_ui_view_only, process_phi_param)
```

Comment:
I think you don't need to import it here as it is not used directly in the file only as `args.database_ui_view_only`